features/step_definitions/web.rb

```diff
@@ -240,3 +240,33 @@
 Given /^I close the current browser$/ do
   browser.finalize
 end
+
+Given /^I check all relatedObjects of clusteroperator "(.*?)" are shown/ do |clusteroperator_name|
+  ensure_admin_tagged
+  #get relatedObjects of clusteroperator
+  relatedObjectsArray = cluster_operator(clusteroperator_name).related_objects
+  relatedObjectsArray.each do |related_object|
+    if related_object['name'] == nil or related_object['name'] == ""
+      next
+    end
```

Comment:
Currently when `status.relatedObjects[N].name` is empty, we will skip to next objects since the object whose name is empty will not be shown on the page